packages/@aws-cdk/aws-eks/lib/shared-interfaces.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * EKS cluster control plane logging configuration
+ */
+export interface ControlPlaneLogging {
+  readonly api?: boolean;
+  readonly audit?: boolean;
+  readonly authenticator?: boolean;
+  readonly controllerManager?: boolean;
+  readonly scheduler?: boolean;
+}
+
+/**
+ * The available cluster control plane log types.
+ */
```

Comment:
This type should not be exported from the module. I think it can just go into cluster-resource.ts